mlir/test/Conversion/AIRToAIE/air_channel_to_locks_ping_pong.mlir

```diff
@@ -262,14 +262,14 @@ func.func @core_to_core_ping_pong() {
 
 // CHECK:    aie.core(%[[VAL_1]])  {
 // CHECK:           aie.use_lock(%[[VAL_3]], AcquireGreaterEqual, 1)
-// CHECK:           aie.use_lock(%[[VAL_4]], Release, 1)
 // CHECK:           aie.use_lock(%[[VAL_3]], AcquireGreaterEqual, 1)
 // CHECK:           aie.use_lock(%[[VAL_4]], Release, 1)
+// CHECK:           aie.use_lock(%[[VAL_4]], Release, 1)
```

Comment:
@erwei-xilinx This makes the test pass but looks wrong. I seem to have broken the core-to-core ping-pong?

Reply:
I agree that this looks wrong... I think we can only know for sure if we try it on hw, but we do not seem to have a board test for core-to-core pingpong for either AIE1 or 2...